bin/MadGraph5_aMCatNLO/cards/production/13TeV/VV/WWTo4Q_5f_amcatnloFxFx/WWTo4Q_5f_amcatnloFxFx_run_card.dat

```diff
@@ -0,0 +1,144 @@
+#***********************************************************************
+#                        MadGraph5_aMC@NLO                             *
+#                                                                      *
+#                      run_card.dat aMC@NLO                            *
+#                                                                      *
+#  This file is used to set the parameters of the run.                 *
+#                                                                      *
+#  Some notation/conventions:                                          *
+#                                                                      *
+#   Lines starting with a hash (#) are info or comments                *
+#                                                                      *
+#   mind the format:   value    = variable     ! comment               *
+#***********************************************************************
+#
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#
+#***********************************************************************
+# Tag name for the run (one word)                                      *
+#***********************************************************************
+  tag_1     = run_tag ! name of the run 
+#***********************************************************************
+# Number of events (and their normalization) and the required          *
+# (relative) accuracy on the Xsec.                                     *
+# These values are ignored for fixed order runs                        *
+#***********************************************************************
+  1000 = nevents ! Number of unweighted events requested 
```

Comment:
should we increase a bit as this is defining the pilot run including madspin. so it might be useful to have a bit more? 5k should take not more time?

Reply:
done